vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,128 @@
+
+ You'll need to run all of these steps before installing Entando the first time but then just steps #1 and #2 after subsequent Windows restarts. 
+
+1. Determine an IP that is unused on your local network. You can use ping or other tools for this but in the following steps we assume that your selected IP is 192.168.99.1.
+
+2. Determine the interface address to Hyper-V, e.g. 32 below. Use cmd `route print` and look for the Interface entry for Hyper-V:
+```
+Interface List
+ 32...00 15 5d 86 45 20 ......Hyper-V Virtual Ethernet Adapter
+```
+
+3. Using elevated privileges, add a persistent route to map your IP to the Hyper-V interface: 
+``` 
+route -p add [YOUR-IP] mask 255.255.255.255 0.0.0.0 IF [HYPER-V-INTERFACE]
+route -p add 192.168.99.1 mask 255.255.255.255 0.0.0.0 IF 32
+```
+4. Verify the route was added by using `route print 192.168.99.1`. This command is useful after restart to check if the route needs to be created again.
+
+5. Next, configure your VM to claim the same address. Shell into the VM using `winpty multipass shell [YOUR-VM-NAME]`. 
+
+6. Change to the root user to make the following steps simpler: `sudo -i`
+
+7. Determine your network adapter via `ip link`, e.g. eth0. You just need the name. It's often second in the list after the loopback adapter.
+``` bash
+ubuntu@primary:~$ ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
+    link/ether 00:15:5d:00:1a:0c brd ff:ff:ff:ff:ff:ff
+```
+
+8. `cd /etc/netplan`
+
+9. Create a netplan entry starting with 0 so it's loaded first: `vi 0-entando.yaml`  
+
+``` yaml
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    [YOUR-NETWORK-ADAPTER]:
+      dhcp4: no
+      addresses: 
+        - [YOUR-IP]/24
+```
+Example:
+
+``` yaml
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    eth0:
+      dhcp4: no
+      addresses: 
+        - 192.168.99.1/24
+```
+
+10. Apply the changes via `netplan apply`
+
+11. Verify connectivity via `ping 192.168.99.1` from the VM. You should get a response rather than a timeout.
+
+12. (Optional) Run a python server to verify you can access the VM from the host at `http://192.168.99.1:8000.` It may take a minute or so before the server is ready.
+
+13. You should now be able to install Entando using the static IP. If your Entando installation stalled during startup and was previously configured using the static IP, it should continue starting up as soon as the external address is functional again. 
+
+#### Option 3: Reinstall Entando
+We're including this option because it works and requires no additional configuration. If you plan to regularly work with Entando, we recommend using Option 1 or 2.
```

Comment:
If you plan to regularly work with Entando  **we recommend developing against a centralized and shared kubernetes instance rather than running a full stack locally.** If you need a cluster locally we recommend using option 1 or 2.

Reply:
Updated.